core/modules/admin_bar/css/admin_bar.css

```diff
@@ -6,10 +6,13 @@
  *
  * @see https://www.htmldog.com/articles/suckerfish
  */
+:root {
+  --admin-bar-height: 2.35rem;
```

Comment:
Is 2.35 (instead of 2.25) a typo? I think, `2.35rem` corresponds to 37.6px, which seems strange to me. Or maybe I missed anything.

Reply:
It's the line height, plus padding. But in rem instead of em/px.

`0,75×1,8+0.5+0.5` which results in these 37.6px height. And that's the actual height of the bar.

This "odd comma" stems from the line height (0,75rem * 1.8 or 12px * 1.8). Plus padding. Did I miss anything?

Reply:
Good to know, thanks for the explanation!